smdx/smdx_64112.xml

```diff
@@ -513,22 +513,22 @@
       <notes></notes>
     </point>
     <point id="CC_Config_DataLog_Daily_AH">
-      <label>Data Log Daily</label>
+      <label>Data Log Daily (Ah)</label>
```

Comment:
I'm used to units going in square brackets like `[Ah]`, but maybe that's a locale specific thing?  Or just me...